app/services/foreman_ansible/inventory_creator.rb

```diff
@@ -95,7 +95,8 @@ def remote_execution_options(host)
         'ansible_user' => host_setting(host, 'remote_execution_ssh_user'),
         'ansible_ssh_pass' => rex_ssh_password(host),
         'ansible_ssh_private_key_file' => ansible_or_rex_ssh_private_key(host),
-        'ansible_port' => host_setting(host, 'remote_execution_ssh_port')
+        'ansible_port' => host_setting(host, 'remote_execution_ssh_port'),
+        'ansible_host' => RemoteExecutionProvider.find_ip_or_hostname(host)
```

Comment:
If we use `find_fqdn` in other places, we don't have to override `find_ip` in `AnsibleProvider` and we could call `AnsibleProvider.find_ip_or_hostname(host)`, which would be less surprising for me to see.